From: unknown@suse.de
Subject: some unknown ieee1394 patch

make the long format the default because its also the default in the
new firewire stack.
Maybe it simplifies migration for new 10.3 installs to 11.0 or later.
Maybe it is bad for existing 10.3 and earlier installs.

modprobe -v sbp2 sbp2_long_sysfs_ieee1394_id=0 to get the old short name
modprobe -v sbp2 sbp2_long_sysfs_ieee1394_id=1 to get the new long name

---
 drivers/ieee1394/sbp2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/ieee1394/sbp2.c
+++ b/drivers/ieee1394/sbp2.c
@@ -225,7 +225,7 @@ MODULE_PARM_DESC(workarounds, "Work arou
  * independent of the implementation of the ieee1394 nodemgr, the longer format
  * is recommended for future use.
  */
-static int sbp2_long_sysfs_ieee1394_id;
+static int sbp2_long_sysfs_ieee1394_id = 1;
 module_param_named(long_ieee1394_id, sbp2_long_sysfs_ieee1394_id, bool, 0644);
 MODULE_PARM_DESC(long_ieee1394_id, "8+3+2 bytes format of ieee1394_id in sysfs "
 		 "(default = backwards-compatible = N, SAM-conforming = Y)");
